### PR TITLE
CLOUDSTACK-8555:Skip testcase for HyperV as it doesn't support volume resize operation

### PR DESCRIPTION
**Pull request by pritisarap12**

### CODE REVIEW

File: test/integration/component/test_browse_volumes.py [hunk 1]
```diff
@@ -2028,6 +2028,10 @@ def test_01_Browser_volume_Life_cycle_tpath(self):
         """
         Test Browser_volume_Life_cycle - This includes upload volume,attach to a VM, write data ,Stop ,Start, Reboot,Reset  of a VM, detach,attach back to the VM, delete volumes  
         """
+        if self.hypervisor.lower() == 'hyperv':
+            self.skipTest("cann't be run for % hypervisor as it doesn't support volume resize operation" % self.hypervisor)
```
Comment (gauravaradhye) [inline test/integration/component/test_browse_volumes.py]: Should be %s instead of %. Change for all applicable places.

**Comment by pritisarap12** [comment]
Done with the review changes.

File: test/integration/component/test_browse_volumes.py [hunk 2]
```diff
@@ -2028,6 +2028,10 @@ def test_01_Browser_volume_Life_cycle_tpath(self):
         """
         Test Browser_volume_Life_cycle - This includes upload volume,attach to a VM, write data ,Stop ,Start, Reboot,Reset  of a VM, detach,attach back to the VM, delete volumes  
         """
+        if self.hypervisor.lower() == 'hyperv':
```
Comment (gauravaradhye) [inline test/integration/component/test_browse_volumes.py]: Can we skip only the resize volume test case with if condition and run test case for other operations?

Comment (pritisarap12) [inline test/integration/component/test_browse_volumes.py]: Yes, we can skip only resize operation and done with modification of the code doing the same.

File: test/integration/component/test_browse_volumes.py [hunk 3]
```diff
@@ -2473,6 +2474,9 @@ def test_11_migrate_upload_volume(self):
         """
         Test Browser_Upload_Volume_migrate_upload_volume
         """
+        if self.hypervisor.lower() == 'hyperv':
```
Comment (gauravaradhye) [inline test/integration/component/test_browse_volumes.py]: Remove this old code snippet which skips the whole test.

**Comment by gauravaradhye** [comment]
LGTM, merging.

**Pull request closed without merging**